src/styles/typography/index.md.njk

```diff
@@ -13,9 +13,14 @@ show_page_nav: true
 
 If your service is on the service.gov.uk subdomain you must use the GDS Transport font.
 
+### When not to use the GDS Transport font
+
 You should use an alternative typeface like Helvetica or Arial for services that are publicly available on different domains.
 
-If you’re not sure whether you should be using GDS Transport, read the Service Manual guide on [making your service look like GOV.UK](https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk) or [contact the Design System team](../../get-in-touch/).
+If you’re not sure whether you should be using GDS Transport, do one of the following:
```

Comment:
Maybe "If you’re not sure whether you should use GDS Transport" just to be slightly more direct and brief?

Reply:
@christopherthomasdesign That makes sense! Thanks.

```suggestion
If you’re not sure whether you should use GDS Transport, do one of the following:
```